test/models/IvdSubmissionSpec.scala

```diff
@@ -76,6 +76,7 @@ class IvdSubmissionSpec extends ModelSpecBase {
     answers <- answers.set(EnterCustomsProcedureCodePage, submission.originalCpc)
     answers <- answers.set(FileUploadPage, submission.supportingDocuments)
     answers <- answers.set(DefermentPage, false)
```

Comment:
This set of user answers should not generate deferment details because of the following line:
```
    answers <- answers.set(DefermentPage, false)
```